### DISCUSSION TITLE
Problem solving a 3D runout model

### DISCUSSION THREAD

**Discussion by cvpontes** [Anura3D/Anura3D_OpenSource#202] [Q&A]
Hello,

I'm trying to model a 3D runout of a stockpile, but when I run it cames out the following message: "Failed finding a new element for particle XXX in element XXX in position XXX". I setted the fixities in the top of the model (z fixed), at the bottom of the model (x y and z fixed or only z fixed), at the right and left directions (x fixed) and at the front and back directions (y fixed). It seems correctly to set like this?

I choosed to calculate the initial stresses by gravity load using the quasi-static method. 

Does anyone has a sugestion to correct the model and make it works?

![Model](https://github.com/Anura3D/Anura3D_OpenSource/assets/163586118/f7e553c8-5182-4115-a452-8cce37a2a927)

Thank you for your attention and help. 

Carla.

**Comment by FrancescaCeccato** [comment]
This error could indicate that there is some sort of numerical instability which should be explored. You have to look into the results and understand where the point is going out of the mesh. Check the material parameters, the conditions and so on. This is a very genreal error and it is not straightforward to identify the cause.
